trafficcloud/app_config.py

```diff
@@ -83,4 +83,44 @@ def check_project_cfg_section(section):
     if section in cfp.sections():  # If the given section exists,
         return True                # then return True.
     else:                          # Otherwise,
-        return False               # then return False
\ No newline at end of file
+        return False               # then return False
+
+def update_tracking_config(config_path, update_dict):
+    """helper function to edit cfg files that look like run_tracking.cfg
+
+    update_dict: i.e. {'nframes': 10, 'video-filename': 'video.avi'}
+    """
+    with open(config_path, 'r') as rf:
+        lines = rf.readlines()
+    with open(config_path, 'w') as wf:
+        for line in lines:
+            line_updated = False
+            line_param = line.split('=')[0].strip()
+            if line_param in update_dict.keys():
+                wf.write("{} = {}\n".format(line_param, update_dict[line_param]))
+                line_updated = True
+            if not line_updated:
+                wf.write(line)
```

Comment:
Good point.  line_updated dance was an artifact from an existing implementation, when I was using a for loop to check if the line_param was in update_dict.keys().  I'll change it.